rails/locale/sv-FI.yml

```diff
@@ -0,0 +1,210 @@
+---
+sv-FI:
+  activerecord:
+    errors:
+      messages:
+        record_invalid: "Ett fel uppstod: %{errors}"
+        restrict_dependent_destroy:
+          has_one: Kan inte ta bort post då beroende %{record} finns
+          has_many: Kan inte ta bort poster då beroende %{record} finns
+  date:
+    abbr_day_names:
+      - sön
+      - mån
+      - tis
+      - ons
+      - tor
+      - fre
+      - lör
+    abbr_month_names:
+      -
+      - jan
+      - feb
+      - mar
+      - apr
+      - maj
+      - jun
+      - jul
+      - aug
+      - sep
+      - okt
+      - nov
+      - dec
+    day_names:
+      - söndag
+      - måndag
+      - tisdag
+      - onsdag
+      - torsdag
+      - fredag
+      - lördag
+    formats:
+      default: "%-d.%-m.%Y"
+      long: "%e %B %Y"
+      short: "%e %b"
+    month_names:
+      -
+      - januari
+      - februari
+      - mars
+      - april
+      - maj
+      - juni
+      - juli
+      - augusti
+      - september
+      - oktober
+      - november
+      - december
+    order:
+      - :day
+      - :month
+      - :year
+  datetime:
+    distance_in_words:
+      about_x_hours:
+        one: ungefär en timme
+        other: ungefär %{count} timmar
+      about_x_months:
+        one: ungefär en månad
+        other: ungefär %{count} månader
+      about_x_years:
+        one: ungefär ett år
+        other: ungefär %{count} år
+      almost_x_years:
+        one: nästan ett år
+        other: nästan %{count} år
+      half_a_minute: en halv minut
+      less_than_x_seconds:
+        one: mindre än en sekund
+        other: mindre än %{count} sekunder
+      less_than_x_minutes:
+        one: mindre än en minut
+        other: mindre än %{count} minuter
+      over_x_years:
+        one: mer än ett år
+        other: mer än %{count} år
+      x_seconds:
+        one: en sekund
+        other: "%{count} sekunder"
+      x_minutes:
+        one: en minut
+        other: "%{count} minuter"
+      x_days:
+        one: en dag
+        other: "%{count} dagar"
+      x_months:
+        one: en månad
+        other: "%{count} månader"
+      x_years:
```

Comment:
I noticed that `x_years` is not present in https://github.com/rails/rails/blob/main/actionview/lib/action_view/locale/en.yml and is present in many of our translations. Is this a copy & paste issue, or I'm missing something?

Reply:
Good find! It’s also not inside the actual helper, so this sounds like something to clean up in rails-i18n.  https://github.com/rails/rails/blob/3661d0d8a46a2725b3e7bacbc230e4f183fb67dc/actionview/lib/action_view/helpers/date_helper.rb#L154

Reply:
Should we change it for `sv-FI` here or separately along all the others in one go?

Reply:
@jolll4 could you compare your proposal to https://github.com/svenfuchs/rails-i18n/tree/master/rails/rails

Let's merge this locale correctly; another time, we will need to correct the remaining locale files. Thanks.